src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ToolTip.cs

```diff
@@ -240,15 +240,15 @@ public async Task When_ToggleButton_DataContext_Set_On_ToolTip_Owner_Nested()
 		[Ignore("Currently fails on macOS, part of #9282 epic")]
 #endif
 		[TestMethod]
-		public Task When_Switch_Theme_UWP() => When_Switch_Theme_Inner(brush => (brush as SolidColorBrush).Color);
+		public Task When_Switch_Theme_Fluent() => When_Switch_Theme_Inner(brush => (brush as SolidColorBrush).Color);
```

Comment:
This test and below one are probably not correct now. We need to swap the lambda passed to `When_Switch_Theme_Inner`